cmd/cli/commands/tasks.go

```diff
@@ -0,0 +1,180 @@
+package commands
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/talis/pkg/api/v1/handlers"
+)
+
+// Task flag names
+const (
+	flagTaskName    = "name" // Reusing "name" constant from projects if applicable, or define separately
+	flagProjectName = "project"
+	flagTaskPage    = "page"
+)
+
+// taskOutput represents the filtered output for a task
+type taskOutput struct {
```

Comment:
[non-blocking] do we want to just commit to returning the full db model to reduce struct management while we are in a period of rapid change? Could be a follow up to update the task and project CLI to remove these custom output types and return all the info. Then the CLI response would match the API response right?